### PR TITLE
Add aju as pgt rep on committee

### PR DESCRIPTION
**Pull request by ajuanijustus**
Added images to 2022-23 assets folders and edited the 2022-23 committee .yaml file

### CODE REVIEW

**Comment by MattyTheHacker** [comment]
I'll do a proper review later, and I can fix this when I merge, but in future commit messages should begin with a capital and be in the imperative tense

**Review by LMBishop — COMMENTED** [review]
Could you scale your mini image to 150x150

File: _data/committee/2022-23.yaml [hunk 1]
```diff
@@ -159,3 +159,12 @@ people:
...
       support and advice. I'd never say no to a few pints and you can usually find me at Joe's bar on a Thursday.
+
+  - name: Aju Ani Justus
+    pronouns: He/Him
+    discordtag: ajewdor#0360
+    role: PGT Rep
+    start-date: 27/10/2022
+    picture: /assets/committee/2022-23/mini/aju.jpg
+    bio: >
+      Hey! I'm a MSc Computer Science student and as PGT Rep I aim to make sure that all postgrad students (on any of the 7 PGT courses) feel welcomed at CSS. Outside of CSS, I enjoy music 🎧, gaming 🎮 and reading high fantasy 🧙‍♂️. Always up for a friendly chat or offering support and advice; feel free to hmu on Discord or LinkedIn, or just saw hi 👋 if you see me around campus!
```
Comment (floofyduckie) [inline _data/committee/2022-23.yaml:170]: Slight typo right at the end, it says "saw" instead of "say"

**Review by floofyduckie — CHANGES_REQUESTED** [review]

File: _data/committee/2022-23.yaml [hunk 2]
```diff
@@ -159,3 +159,12 @@ people:
...
       support and advice. I'd never say no to a few pints and you can usually find me at Joe's bar on a Thursday.
+
+  - name: Aju Ani Justus
+    pronouns: He/Him
+    discordtag: ajewdor#0360
+    role: PGT Rep
+    start-date: 27/10/2022
+    picture: /assets/committee/2022-23/mini/aju.jpg
+    bio: >
+      Hey! I'm a MSc Computer Science student and as PGT Rep I aim to make sure that all postgrad students (on any of the 7 PGT courses) feel welcomed at CSS. Outside of CSS, I enjoy music 🎧, gaming 🎮 and reading high fantasy 🧙‍♂️. Always up for a friendly chat or offering support and advice; feel free to hmu on Discord or LinkedIn, or say hi 👋 if you see me around campus!
```
Comment (floofyduckie) [inline _data/committee/2022-23.yaml:170]: looks good! :)

**Review by floofyduckie — APPROVED** [review]

**Review by JTWWilson — APPROVED** [review]

**Review by sapphyree — APPROVED** [review]
LGTM

**Review by MattyTheHacker — APPROVED** [review]

**Pull request merged**